src/main/java/th/ac/kmitl/science/comsci/example/controllers/HelloServlet.java

```diff
@@ -23,6 +23,8 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         out.println("<ul>");
         out.println("<li>Bearchitect</li>");
         out.println("<li>58050237 Chayapol Limanon</li>");
+	out.println("<li>58050231 Patharachat Chanakitkumjorn</li>");
```

Comment:
please pull upstream to merge for "57050294 Pichavarit Panyokaew" and "58050276 Thanathat Surakhup" and adjust the alignment on line 26